File: ViewModelsLayer/ViewModels/HistoryViewModels/HistoryUserRoundViewModel.cs
using EntitiesLayer.Enums;
using System.Collections.Generic;
using ViewModelsLayer.ViewModels.CardViewModels;

namespace ViewModelsLayer.ViewModels.HistoryViewModels
{
    public class HistoryUserRoundViewModel : UserBaseViewModel
    {
        public int Points { get; set; }
        public RoundStatus RoundStatus { get; set; }
        public List<ResponseCardViewModel> Cards { get; set; }
    }
}

File: API/Angular/src/app/viewmodels/HistoryViewModels/history-user-round-view-model.ts
import { UserRole } from 'src/app/shared/enums/user-role';
import { RoundStatus } from 'src/app/shared/enums/round-status';
import { ResponseCardViewModel } from '../CardViewModels/response-card-view-model';

export default class HistoryUserRoundViewModel
{
  constructor(userId, nickname, userRole, points, roundStatus, cards)
  {
    this.userId = userId;
    this.nickname = nickname;
    this.userRole = userRole;
    this.points = points;
    this.roundStatus = roundStatus;
    this.cards = cards;
  }

  public userId: number;
  public nickname: string;
  public userRole: UserRole;
  public points: number;
  public roundStatus: RoundStatus;
  public cards: Array<ResponseCardViewModel>;
}
